### PR TITLE
Translate variant + add check delete

### PR DESCRIPTION
**Pull request by Delphineray**

### CODE REVIEW

File: content/md/first-steps/manage-families.md [hunk 1]
```diff
@@ -268,6 +278,18 @@ To view the family versions:
 Deleting a family does not delete the attributes it includes, or the products associated with it. The attributes are always available and the products keep these very same attributes if they have values: they will become “optional” and the products with no family will have no longer completeness. The values are though maintained: there is no loss of product information.
 :::
 
+::: warning
+A family with family variants cannot be deleted. You need to first [delete its family variants](/articles/manage-your-families.html#delete-a-family-variant-since-22).
```
Comment (KmilleF) [inline content/md/first-steps/manage-families.md]: AFAIK, we should say "you first need to"

File: content/md/first-steps/manage-families.md [hunk 2]
```diff
@@ -268,6 +278,18 @@ To view the family versions:
...
+:::
+
+::: warning
+_Since 2.3_, a family cannot be deleted if there are some products with this family, you need to first remove the products from this family (refer to [the bulk action change family of products](/articles/product-mass-actions.html)).
```
Comment (KmilleF) [inline content/md/first-steps/manage-families.md]: "if there are some products within this family"?

File: content/md/first-steps/manage-families.md (hunk 1 above)
Comment (Delphineray) [reply content/md/first-steps/manage-families.md]: ah yes :)

File: content/md/first-steps/manage-families.md (hunk 2 above)
Comment (Delphineray) [reply content/md/first-steps/manage-families.md]: yep :)

**Pull request merged**